tests/shared/src/test/scala/scala/meta/tests/prettyprinters/SyntacticSuite.scala

```diff
@@ -1087,6 +1087,10 @@ class SyntacticSuite extends scala.meta.tests.parsers.ParseSuite {
     checkTree(Lit.String("ラーメン"), "\"ラーメン\"")
   }
 
+  test("#2447 Pat.Bind on Term.Name") {
+    checkTree(q"{ case x @ Y => x }", "{\n  case x @ Y => x\n}")
+  }
```

Comment:
Could you also add a test with object name in backtics?
```scala
checkTree(q"{ case x @ `y` => x }", "{\n case x @ `y` => x\n}")
```

Reply:
Added!